src/components/common/TermCard/TermCard.js

```diff
@@ -33,4 +35,13 @@ const TermCard = ({ term }) => (
     </div>
 );
 
+TermCard.propTypes = {
+    term: PropTypes.shape({
+        entry: PropTypes.string,
+        meaning: PropTypes.meaning,
+        type: PropTypes.string,
+        examples: PropTypes.string,
```

Comment:
examples deve ser uma lista de string
```suggestion
        examples: PropTypes.arrayOf(PropTypes.string),
```

Reply:
adciona a prop acronym também
`acronym: Proptypes.string`